common/cmake/sv2v.cmake

```diff
@@ -0,0 +1,87 @@
+# This CMake include defines the following functions:
+#
+# * ADD_SV2V_TARGET - Converts SystemVerilog sources to Verilog using sv2v
+
+function(ADD_SV2V_TARGET)
+  # ~~~
+  # ADD_SV2V_TARGET(
+  #    NAME <name>
+  #    SOURCES <source list>
+  #    INCLUDES <include list>
+  #    FLAGS <flags>
+  #    [EXPLICIT_ADD_FILE_TARGET]
+  #
+  # ADD_SV2V_TARGET is used for converting SystemVerilog sources to Verilog.
+  # SOURCES will be passed to the conversion tool, whereas INCLUDES will be
+  # just copied to the build directory.
+  # By default SOURCES and INCLUDES will be implicitly passed to ADD_FILE_TARGET.
+  # If EXPLICIT_ADD_FILE_TARGET is supplied, this behavior is suppressed.
+  #
+  # FLAGS is a string with the flags that are passed to the sv2v during the conversion.
+  #
+  # Targets  generated:
+  # * <name>_sv2v - Output sv2v file in a Verilog format.
+  #
+  # Output files:
+  #
+  # * <name>_sv2v.v - Output Verilog file
+  # * <name>_sv2v.v.log - Log file with errors and warnings during the sv2v conversion
+  #
+  # Outputs for this target will be located in the ${CMAKE_CURRENT_BINARY_DIR}
+  # ~~~
+
+  set(options EXPLICIT_ADD_FILE_TARGET)
```

Comment:
We should not add "EXPLICIT_ADD_FILE_TARGET" to new commands.

Reply:
Removed